samples/Client/DaprClient/Program.cs

```diff
@@ -82,10 +82,31 @@ internal static async Task InvokeMethodAsync(DaprClient client)
             Console.WriteLine("Deleted State!");
         }
 
+        // This example shows how to invoke a method on a service listening on http.
+        // In such a scenario, a key value pair of http.verb and the verb must be added to the metadata parameter.
+        internal static async Task InvokeMethodOnHttpServiceAsync(DaprClient client)
+        {
+            MyData data = new MyData() { Message = "mydata" };
+            Dictionary<string, string> m = new Dictionary<string, string>();
+            m.Add("http.verb", "POST");
```

Comment:
Isn't POST the default for invoking methods? It's a wierd sample to just show that it's possible to do what dapr does by default 😆

Reply:
yes, will change it to use GET verb and mention that POST is default.